components/org.wso2.carbon.identity.password.expiry/src/main/java/org/wso2/carbon/identity/password/expiry/models/PasswordExpiryRule.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.password.expiry.models;
+
+import org.apache.commons.lang.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class to represent a password expiry rule.
+ */
+public class PasswordExpiryRule {
+
+    private int priority;
+    private int expiryDays;
+    private PasswordExpiryRuleAttributeEnum attribute;
+    private PasswordExpiryRuleOperatorEnum operator;
+    private List<String> values;
```

Comment:
We can define empty arrayList instead of setting 
this.values = new ArrayList<>(); later.

Reply:
Added